seaborn/categorical.py

```diff
@@ -2032,14 +2027,13 @@ def draw_letter_value_plot(self, ax, kws):
                     self._lvplot(box_data,
                                  positions=[center],
                                  color=color,
-                                 vert=vert,
                                  widths=self.nested_width,
-                                 k_depth=self.k_depth,
                                  ax=ax,
-                                 scale=self.scale,
-                                 outlier_prop=self.outlier_prop,
                                  **kws)
 
+        # Autoscale the values axis to make sure all patches are visible
+        ax.autoscale_view(scalex=self.orient == "h", scaley=self.orient == "v")
```

Comment:
Is this sufficient or do we need to autoscale on both dimensions now? I can't remember which of the recent changes in matplotlib were intentional and which were bugs.

Reply:
There is a relevant issue in swarmplot I am thinking of.

Reply:
I reckon that calling `ax.plot` for plotting the medians takes care of the width dimension.